vendor.conf

```diff
@@ -54,7 +54,7 @@ golang.org/x/sync                                   42b317875d0fa942474b76e1b46a
 golang.org/x/sys                                    5c8b2ff67527cb88b770f693cebf3799036d8bc0
 golang.org/x/text                                   19e51611da83d6be54ddafce4a4af510cb3e9ea4
 google.golang.org/genproto                          e50cd9704f63023d62cd06a1994b98227fc4d21a
-google.golang.org/grpc                              v1.27.1
+google.golang.org/grpc                              v1.26.0
```

Comment:
This is a dependency for https://github.com/containerd/containerd, and rolled back to an older version (which was updated to this version in https://github.com/containerd/containerd/pull/4053); did the newer version not work?

Reply:
Newer version is fine when i tried it out earlier. will just stick to 1.27.1